src/web/src/security_removeheaders.ecpp

```diff
@@ -0,0 +1,48 @@
+<#
+ #
+ # Copyright (C) 2018 Eaton
+ #
+ # This program is free software; you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation; either version 2 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License along
+ # with this program; if not, write to the Free Software Foundation, Inc.,
+ # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ #
+ #><#
+/*!
+ * \file security_removeheaders.ecpp
+ * example : remove Cache-Control from HTTP headers for all .ttf image files
+ * <mapping>
+ *     <target>security_removeheaders@bios</target>
+ *     <url>\.ttf$</url>
+ *     <args>
+ *         <remove>Cache-Control</remove>
+ *     </args>
+ *   </mapping>
+ * \author Gerald Guillaume <GeraldGuillaume@Eaton.com>
+ * \brief  remove HTTP headers defined as args in tntnet.xml
+ */
+#><%pre>
```

Comment:
Why do you need to comment this line? Isn't the <%pre> needed by the </%pre> 2 lines below?